app/assets/javascripts/widgets/rss_feed_display.js

```diff
@@ -93,13 +93,13 @@ $(document).ready(function() {
       $feedItemDescription.className          = 'rss-feed-item__description-text';
       $feedItemLink.className                 = 'rss-feed-item__link text--link';
       $feedItemDateContainer.className        = 'rss-feed-item__date-container';
+      $feedItemDate.className                 = 'rss-feed-item__date text__bold';
+      $feedItemTime.className                 = 'rss-feed-item__time';
 
       // ADDED INNER TEXT, DATE, AND TIME TO APPOPRIATE RSS DISPLAY ELEMENTS
```

Comment:
Did this small cleanup I wanted to make to the rss feed after I closed and deleted that story/branch